<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>1.2 Posters | eRum2020 Contributed Program</title>
  <meta name="description" content="1.2 Posters | eRum2020 Contributed Program" />
  <meta name="generator" content="bookdown 0.20 and GitBook 2.6.7" />

  <meta property="og:title" content="1.2 Posters | eRum2020 Contributed Program" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="1.2 Posters | eRum2020 Contributed Program" />
  
  
  

<meta name="author" content="eRum2020 organizing Committee" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="lightning-talks.html"/>
<link rel="next" href="regular-talks.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> eRum2020 Contributed Program</a><ul>
<li class="chapter" data-level="1.1" data-path="lightning-talks.html"><a href="lightning-talks.html"><i class="fa fa-check"></i><b>1.1</b> Lightning talks</a><ul>
<li class="chapter" data-level="1.1.1" data-path="lightning-talks.html"><a href="lightning-talks.html#an-enriched-disease-risk-assessment-model-based-on-historical-blood-donors-records"><i class="fa fa-check"></i><b>1.1.1</b> An enriched disease risk assessment model based on historical blood donors records</a></li>
<li class="chapter" data-level="1.1.2" data-path="lightning-talks.html"><a href="lightning-talks.html#rdwd-r-interface-to-german-weather-service-data"><i class="fa fa-check"></i><b>1.1.2</b> rdwd: R interface to German Weather Service data</a></li>
<li class="chapter" data-level="1.1.3" data-path="lightning-talks.html"><a href="lightning-talks.html#tv-show-data-frames-in-the-browser"><i class="fa fa-check"></i><b>1.1.3</b> tv: Show Data Frames in the Browser</a></li>
<li class="chapter" data-level="1.1.4" data-path="lightning-talks.html"><a href="lightning-talks.html#predicting-the-euro-2020-results-using-tournament-rank-probabilities-scores-from-the-soccer-package"><i class="fa fa-check"></i><b>1.1.4</b> Predicting the Euro 2020 results using tournament rank probabilities scores from the socceR package</a></li>
<li class="chapter" data-level="1.1.5" data-path="lightning-talks.html"><a href="lightning-talks.html#differential-enriched-scan-2-descan2-an-r-pipeline-for-epigenomic-analysis."><i class="fa fa-check"></i><b>1.1.5</b> Differential Enriched Scan 2 (DEScan2): an R pipeline for epigenomic analysis.</a></li>
<li class="chapter" data-level="1.1.6" data-path="lightning-talks.html"><a href="lightning-talks.html#ultra-fast-penalized-regressions-with-r-package-bigstatsr"><i class="fa fa-check"></i><b>1.1.6</b> Ultra fast penalized regressions with R package {bigstatsr}</a></li>
<li class="chapter" data-level="1.1.7" data-path="lightning-talks.html"><a href="lightning-talks.html#supporting-twitter-analytics-application-with-graph-databases-and-the-arangodb-package"><i class="fa fa-check"></i><b>1.1.7</b> Supporting Twitter analytics application with graph-databases and the aRangodb package</a></li>
<li class="chapter" data-level="1.1.8" data-path="lightning-talks.html"><a href="lightning-talks.html#reproducible-data-visualization-with-canvasxpress"><i class="fa fa-check"></i><b>1.1.8</b> Reproducible Data Visualization with CanvasXpress</a></li>
<li class="chapter" data-level="1.1.9" data-path="lightning-talks.html"><a href="lightning-talks.html#design-your-own-quantum-simulator-with-r"><i class="fa fa-check"></i><b>1.1.9</b> Design your own quantum simulator with R</a></li>
<li class="chapter" data-level="1.1.10" data-path="lightning-talks.html"><a href="lightning-talks.html#what-are-the-potato-eaters-eating"><i class="fa fa-check"></i><b>1.1.10</b> What are the potato eaters eating</a></li>
<li class="chapter" data-level="1.1.11" data-path="lightning-talks.html"><a href="lightning-talks.html#dm-working-with-relational-data-models-in-r"><i class="fa fa-check"></i><b>1.1.11</b> dm: working with relational data models in R</a></li>
<li class="chapter" data-level="1.1.12" data-path="lightning-talks.html"><a href="lightning-talks.html#explaining-black-box-models-with-xspliner-to-make-deliberate-business-decisions"><i class="fa fa-check"></i><b>1.1.12</b> Explaining black-box models with xspliner to make deliberate business decisions</a></li>
<li class="chapter" data-level="1.1.13" data-path="lightning-talks.html"><a href="lightning-talks.html#using-open-access-data-to-derive-genome-composition-of-emerging-viruses"><i class="fa fa-check"></i><b>1.1.13</b> Using open-access data to derive genome composition of emerging viruses</a></li>
<li class="chapter" data-level="1.1.14" data-path="lightning-talks.html"><a href="lightning-talks.html#a-principal-component-analysis-based-method-to-detect-biomarker-captation-from-vibrational-spectra"><i class="fa fa-check"></i><b>1.1.14</b> A principal component analysis based method to detect biomarker captation from vibrational spectra</a></li>
<li class="chapter" data-level="1.1.15" data-path="lightning-talks.html"><a href="lightning-talks.html#an-innovative-way-to-support-your-sales-force"><i class="fa fa-check"></i><b>1.1.15</b> An innovative way to support your sales force</a></li>
<li class="chapter" data-level="1.1.16" data-path="lightning-talks.html"><a href="lightning-talks.html#ptmixed-an-r-package-for-flexible-modelling-of-longitudinal-overdispersed-count-data"><i class="fa fa-check"></i><b>1.1.16</b> ptmixed: an R package for flexible modelling of longitudinal overdispersed count data</a></li>
<li class="chapter" data-level="1.1.17" data-path="lightning-talks.html"><a href="lightning-talks.html#one-way-non-normal-anova-in-reliability-analysis-using-with-doex"><i class="fa fa-check"></i><b>1.1.17</b> One-way non-normal ANOVA in reliability analysis using with doex</a></li>
<li class="chapter" data-level="1.1.18" data-path="lightning-talks.html"><a href="lightning-talks.html#keeping-on-top-of-r-in-real-time-high-stakes-trading-systems"><i class="fa fa-check"></i><b>1.1.18</b> Keeping on top of R in Real-Time, High-Stakes trading systems</a></li>
<li class="chapter" data-level="1.1.19" data-path="lightning-talks.html"><a href="lightning-talks.html#towards-more-structured-data-quality-assessment-in-the-process-mining-field-the-daqapo-package"><i class="fa fa-check"></i><b>1.1.19</b> Towards more structured data quality assessment in the process mining field: the DaQAPO package</a></li>
<li class="chapter" data-level="1.1.20" data-path="lightning-talks.html"><a href="lightning-talks.html#analyzing-preference-data-with-the-bayesmallows-package"><i class="fa fa-check"></i><b>1.1.20</b> Analyzing Preference Data with the BayesMallows Package</a></li>
<li class="chapter" data-level="1.1.21" data-path="lightning-talks.html"><a href="lightning-talks.html#predicting-business-cycle-fluctuations-using-text-analytics"><i class="fa fa-check"></i><b>1.1.21</b> Predicting Business Cycle Fluctuations Using Text Analytics</a></li>
<li class="chapter" data-level="1.1.22" data-path="lightning-talks.html"><a href="lightning-talks.html#flexible-deep-learning-via-the-juliaconnector"><i class="fa fa-check"></i><b>1.1.22</b> Flexible deep learning via the JuliaConnectoR</a></li>
<li class="chapter" data-level="1.1.23" data-path="lightning-talks.html"><a href="lightning-talks.html#time-series-missing-data-visualizations"><i class="fa fa-check"></i><b>1.1.23</b> Time Series Missing Data Visualizations</a></li>
<li class="chapter" data-level="1.1.24" data-path="lightning-talks.html"><a href="lightning-talks.html#effectclass-an-r-package-to-interpret-effects-and-visualise-uncertainty"><i class="fa fa-check"></i><b>1.1.24</b> effectclass: an R package to interpret effects and visualise uncertainty</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="posters.html"><a href="posters.html"><i class="fa fa-check"></i><b>1.2</b> Posters</a><ul>
<li class="chapter" data-level="1.2.1" data-path="posters.html"><a href="posters.html#a-flexible-dashboard-for-monitoring-platform-trials"><i class="fa fa-check"></i><b>1.2.1</b> A flexible dashboard for monitoring platform trials</a></li>
<li class="chapter" data-level="1.2.2" data-path="posters.html"><a href="posters.html#prda-package-enhancing-statistical-inference-via-prospective-and-retrospective-design-analysis."><i class="fa fa-check"></i><b>1.2.2</b> PRDA package: Enhancing Statistical Inference via Prospective and Retrospective Design Analysis.</a></li>
<li class="chapter" data-level="1.2.3" data-path="posters.html"><a href="posters.html#automate-flexdashboard-with-github"><i class="fa fa-check"></i><b>1.2.3</b> Automate flexdashboard with GitHub</a></li>
<li class="chapter" data-level="1.2.4" data-path="posters.html"><a href="posters.html#easyreporting-a-bioconductor-package-for-reproducible-research-implementation"><i class="fa fa-check"></i><b>1.2.4</b> EasyReporting: a Bioconductor package for Reproducible Research implementation</a></li>
<li class="chapter" data-level="1.2.5" data-path="posters.html"><a href="posters.html#newwave-a-scalable-r-package-for-the-dimensionality-reduction-of-single-cell-rna-seq"><i class="fa fa-check"></i><b>1.2.5</b> NewWave: a scalable R package for the dimensionality reduction of single-cell RNA-seq</a></li>
<li class="chapter" data-level="1.2.6" data-path="posters.html"><a href="posters.html#orf-ordered-random-forests"><i class="fa fa-check"></i><b>1.2.6</b> orf: Ordered Random Forests</a></li>
<li class="chapter" data-level="1.2.7" data-path="posters.html"><a href="posters.html#power-supply-health-status-monitoring-dashboard"><i class="fa fa-check"></i><b>1.2.7</b> Power Supply health status monitoring dashboard</a></li>
<li class="chapter" data-level="1.2.8" data-path="posters.html"><a href="posters.html#first-year-ict-students-dropout-predicting-with-r-models"><i class="fa fa-check"></i><b>1.2.8</b> First-year ICT students dropout predicting with R models</a></li>
<li class="chapter" data-level="1.2.9" data-path="posters.html"><a href="posters.html#benchmark-percentage-disjoint-data-splitting-in-cross-validation-for-assessing-the-skill-of-machine"><i class="fa fa-check"></i><b>1.2.9</b> Benchmark Percentage Disjoint Data Splitting in Cross Validation for Assessing the Skill of Machine</a></li>
<li class="chapter" data-level="1.2.10" data-path="posters.html"><a href="posters.html#integrating-professional-software-engineering-practices-in-medical-research-software"><i class="fa fa-check"></i><b>1.2.10</b> Integrating professional software engineering practices in medical research software</a></li>
<li class="chapter" data-level="1.2.11" data-path="posters.html"><a href="posters.html#dealing-with-changing-administrative-boundaries-the-case-of-swiss-municipalities"><i class="fa fa-check"></i><b>1.2.11</b> Dealing with changing administrative boundaries: The case of Swiss municipalities</a></li>
<li class="chapter" data-level="1.2.12" data-path="posters.html"><a href="posters.html#baddea-an-r-package-for-measuring-firms-efficiency-adjusted-by-undesirable-outputs"><i class="fa fa-check"></i><b>1.2.12</b> badDEA: An R package for measuring firms’ efficiency adjusted by undesirable outputs</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="regular-talks.html"><a href="regular-talks.html"><i class="fa fa-check"></i><b>1.3</b> Regular talks</a><ul>
<li class="chapter" data-level="1.3.1" data-path="regular-talks.html"><a href="regular-talks.html#design-patterns-for-big-shiny-apps"><i class="fa fa-check"></i><b>1.3.1</b> Design Patterns For Big Shiny Apps</a></li>
<li class="chapter" data-level="1.3.2" data-path="regular-talks.html"><a href="regular-talks.html#using-xgboost-plumber-and-docker-in-production-to-power-a-new-banking-product"><i class="fa fa-check"></i><b>1.3.2</b> Using XGBoost, Plumber and Docker in production to power a new banking product</a></li>
<li class="chapter" data-level="1.3.3" data-path="regular-talks.html"><a href="regular-talks.html#astronomical-source-detection-and-background-separation-a-bayesian-nonparametric-approach"><i class="fa fa-check"></i><b>1.3.3</b> Astronomical source detection and background separation: a Bayesian nonparametric approach</a></li>
<li class="chapter" data-level="1.3.4" data-path="regular-talks.html"><a href="regular-talks.html#creating-drag-and-drop-shiny-applications-using-sortable"><i class="fa fa-check"></i><b>1.3.4</b> Creating drag-and-drop shiny applications using sortable</a></li>
<li class="chapter" data-level="1.3.5" data-path="regular-talks.html"><a href="regular-talks.html#high-dimensional-sampling-and-volume-computation"><i class="fa fa-check"></i><b>1.3.5</b> High dimensional sampling and volume computation</a></li>
<li class="chapter" data-level="1.3.6" data-path="regular-talks.html"><a href="regular-talks.html#fake-news-ai-on-the-battle-ground"><i class="fa fa-check"></i><b>1.3.6</b> Fake News: AI on the battle ground</a></li>
<li class="chapter" data-level="1.3.7" data-path="regular-talks.html"><a href="regular-talks.html#from-consulting-to-open-source-and-back"><i class="fa fa-check"></i><b>1.3.7</b> From consulting to open-source and back</a></li>
<li class="chapter" data-level="1.3.8" data-path="regular-talks.html"><a href="regular-talks.html#deduplicating-real-estate-ads-using-naive-bayes-record-linkage"><i class="fa fa-check"></i><b>1.3.8</b> Deduplicating real estate ads using Naive Bayes record linkage</a></li>
<li class="chapter" data-level="1.3.9" data-path="regular-talks.html"><a href="regular-talks.html#polite-web-etiquette-for-r-users"><i class="fa fa-check"></i><b>1.3.9</b> {polite}: web etiquette for R users</a></li>
<li class="chapter" data-level="1.3.10" data-path="regular-talks.html"><a href="regular-talks.html#hydrological-modelling-and-r"><i class="fa fa-check"></i><b>1.3.10</b> Hydrological Modelling and R</a></li>
<li class="chapter" data-level="1.3.11" data-path="regular-talks.html"><a href="regular-talks.html#genetonic-enjoy-rna-seq-data-analysis-responsibly"><i class="fa fa-check"></i><b>1.3.11</b> GeneTonic: enjoy RNA-seq data analysis, responsibly</a></li>
<li class="chapter" data-level="1.3.12" data-path="regular-talks.html"><a href="regular-talks.html#a-simple-and-flexible-inactivitysleep-detection-r-package"><i class="fa fa-check"></i><b>1.3.12</b> A simple and flexible inactivity/sleep detection R package</a></li>
<li class="chapter" data-level="1.3.13" data-path="regular-talks.html"><a href="regular-talks.html#progressr-an-inclusive-unifying-api-for-progress-updates"><i class="fa fa-check"></i><b>1.3.13</b> progressr: An Inclusive, Unifying API for Progress Updates</a></li>
<li class="chapter" data-level="1.3.14" data-path="regular-talks.html"><a href="regular-talks.html#varycoef-modeling-spatially-varying-coefficients"><i class="fa fa-check"></i><b>1.3.14</b> varycoef: Modeling Spatially Varying Coefficients</a></li>
<li class="chapter" data-level="1.3.15" data-path="regular-talks.html"><a href="regular-talks.html#fastai-in-r-preserving-wildlife-with-computer-vision"><i class="fa fa-check"></i><b>1.3.15</b> FastAI in R: preserving wildlife with computer vision</a></li>
<li class="chapter" data-level="1.3.16" data-path="regular-talks.html"><a href="regular-talks.html#the-r-consortium-2020-adapting-to-rapid-change-and-global-crisis"><i class="fa fa-check"></i><b>1.3.16</b> The R Consortium 2020: adapting to rapid change and global crisis</a></li>
<li class="chapter" data-level="1.3.17" data-path="regular-talks.html"><a href="regular-talks.html#powering-turing-e-atlas-with-r"><i class="fa fa-check"></i><b>1.3.17</b> Powering Turing e-Atlas with R</a></li>
<li class="chapter" data-level="1.3.18" data-path="regular-talks.html"><a href="regular-talks.html#using-process-mining-principles-to-extract-a-collaboration-graph-from-a-version-control-system-log"><i class="fa fa-check"></i><b>1.3.18</b> Using process mining principles to extract a collaboration graph from a version control system log</a></li>
<li class="chapter" data-level="1.3.19" data-path="regular-talks.html"><a href="regular-talks.html#manifoldgstat-an-r-package-for-spatial-statistics-of-manifold-data"><i class="fa fa-check"></i><b>1.3.19</b> Manifoldgstat: an R package for spatial statistics of manifold data</a></li>
<li class="chapter" data-level="1.3.20" data-path="regular-talks.html"><a href="regular-talks.html#voronoi-linkage-for-spatially-misaligned-data"><i class="fa fa-check"></i><b>1.3.20</b> Voronoi Linkage for Spatially Misaligned Data</a></li>
<li class="chapter" data-level="1.3.21" data-path="regular-talks.html"><a href="regular-talks.html#be-proud-of-your-code-tools-and-patterns-for-making-production-ready-clean-r-code"><i class="fa fa-check"></i><b>1.3.21</b> Be proud of your code! Tools and patterns for making production-ready, clean R code</a></li>
<li class="chapter" data-level="1.3.22" data-path="regular-talks.html"><a href="regular-talks.html#going-in-the-fast-lane-with-r.-how-we-use-r-within-the-biggest-digital-dealer-program-in-emea."><i class="fa fa-check"></i><b>1.3.22</b> Going in the fast lane with R. How we use R within the biggest digital dealer program in EMEA.</a></li>
<li class="chapter" data-level="1.3.23" data-path="regular-talks.html"><a href="regular-talks.html#r-alongside-airflow-docker-and-gitlab-ci"><i class="fa fa-check"></i><b>1.3.23</b> R alongside Airflow, Docker and Gitlab CI</a></li>
<li class="chapter" data-level="1.3.24" data-path="regular-talks.html"><a href="regular-talks.html#damirseq-2.0-from-high-dimensional-data-to-cost-effective-reliable-prediction-models"><i class="fa fa-check"></i><b>1.3.24</b> DaMiRseq 2.0: from high dimensional data to cost-effective reliable prediction models</a></li>
<li class="chapter" data-level="1.3.25" data-path="regular-talks.html"><a href="regular-talks.html#how-to-apply-r-in-a-hospital-environment-on-standard-available-hospital-wide-data"><i class="fa fa-check"></i><b>1.3.25</b> How to apply R in a hospital environment on standard available hospital-wide data</a></li>
<li class="chapter" data-level="1.3.26" data-path="regular-talks.html"><a href="regular-talks.html#computer-algebra-systems-in-r"><i class="fa fa-check"></i><b>1.3.26</b> Computer Algebra Systems in R</a></li>
<li class="chapter" data-level="1.3.27" data-path="regular-talks.html"><a href="regular-talks.html#interpretable-and-accessible-deep-learning-for-omics-data-with-r-and-friends"><i class="fa fa-check"></i><b>1.3.27</b> Interpretable and accessible Deep Learning for omics data with R and friends</a></li>
<li class="chapter" data-level="1.3.28" data-path="regular-talks.html"><a href="regular-talks.html#elevating-shiny-module-with-tidymodules"><i class="fa fa-check"></i><b>1.3.28</b> Elevating shiny module with {tidymodules}</a></li>
<li class="chapter" data-level="1.3.29" data-path="regular-talks.html"><a href="regular-talks.html#apfr-average-power-function-and-bayes-fdr-for-robust-brain-networks-construction"><i class="fa fa-check"></i><b>1.3.29</b> APFr: Average Power Function and Bayes FDR for Robust Brain Networks Construction</a></li>
<li class="chapter" data-level="1.3.30" data-path="regular-talks.html"><a href="regular-talks.html#flexible-meta-analysis-of-generalized-additive-models-with-metagam"><i class="fa fa-check"></i><b>1.3.30</b> Flexible Meta-Analysis of Generalized Additive Models with metagam</a></li>
<li class="chapter" data-level="1.3.31" data-path="regular-talks.html"><a href="regular-talks.html#epimod-a-computational-framework-for-studying-epidemiological-systems."><i class="fa fa-check"></i><b>1.3.31</b> EPIMOD: A computational framework for studying epidemiological systems.</a></li>
<li class="chapter" data-level="1.3.32" data-path="regular-talks.html"><a href="regular-talks.html#correlaidx---building-r-focused-communities-for-social-good-on-the-local-level"><i class="fa fa-check"></i><b>1.3.32</b> CorrelAidX - Building R-focused Communities for Social Good on the Local Level</a></li>
<li class="chapter" data-level="1.3.33" data-path="regular-talks.html"><a href="regular-talks.html#interactive-visualization-of-complex-texts"><i class="fa fa-check"></i><b>1.3.33</b> Interactive visualization of complex texts</a></li>
<li class="chapter" data-level="1.3.34" data-path="regular-talks.html"><a href="regular-talks.html#connector-a-computational-approach-to-study-intratumor-heterogeneity."><i class="fa fa-check"></i><b>1.3.34</b> CONNECTOR: a computational approach to study intratumor heterogeneity.</a></li>
<li class="chapter" data-level="1.3.35" data-path="regular-talks.html"><a href="regular-talks.html#gwqs-an-r-package-for-linear-and-generalized-weighted-quantile-sum-wqs-regression"><i class="fa fa-check"></i><b>1.3.35</b> gWQS: An R Package for Linear and Generalized Weighted Quantile Sum (WQS) Regression</a></li>
<li class="chapter" data-level="1.3.36" data-path="regular-talks.html"><a href="regular-talks.html#transparent-journalism-through-the-power-of-r"><i class="fa fa-check"></i><b>1.3.36</b> Transparent Journalism Through the Power of R</a></li>
<li class="chapter" data-level="1.3.37" data-path="regular-talks.html"><a href="regular-talks.html#whats-new-in-shinyproxy"><i class="fa fa-check"></i><b>1.3.37</b> What’s New in ShinyProxy</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="shiny-demos.html"><a href="shiny-demos.html"><i class="fa fa-check"></i><b>1.4</b> Shiny demos</a><ul>
<li class="chapter" data-level="1.4.1" data-path="shiny-demos.html"><a href="shiny-demos.html#visualising-and-modelling-bike-sharing-mobility-usage-in-the-city-of-milan"><i class="fa fa-check"></i><b>1.4.1</b> Visualising and Modelling Bike Sharing Mobility usage in the city of Milan</a></li>
<li class="chapter" data-level="1.4.2" data-path="shiny-demos.html"><a href="shiny-demos.html#media-shiny-marketing-mix-models-builder"><i class="fa fa-check"></i><b>1.4.2</b> Media Shiny: Marketing Mix Models Builder</a></li>
<li class="chapter" data-level="1.4.3" data-path="shiny-demos.html"><a href="shiny-demos.html#espres-a-shiny-web-tool-to-support-river-basin-management-planning-in-european-watersheds"><i class="fa fa-check"></i><b>1.4.3</b> ESPRES: A shiny web tool to support River Basin Management planning in European Watersheds</a></li>
<li class="chapter" data-level="1.4.4" data-path="shiny-demos.html"><a href="shiny-demos.html#tsviz-a-data-scientist-friendly-addin-for-rstudio"><i class="fa fa-check"></i><b>1.4.4</b> tsviz: a data-scientist-friendly addin for RStudio</a></li>
<li class="chapter" data-level="1.4.5" data-path="shiny-demos.html"><a href="shiny-demos.html#mobility-scan"><i class="fa fa-check"></i><b>1.4.5</b> Mobility scan</a></li>
<li class="chapter" data-level="1.4.6" data-path="shiny-demos.html"><a href="shiny-demos.html#developing-shiny-applications-to-facilitate-precision-agriculture-workflows"><i class="fa fa-check"></i><b>1.4.6</b> Developing Shiny applications to facilitate precision agriculture workflows</a></li>
<li class="chapter" data-level="1.4.7" data-path="shiny-demos.html"><a href="shiny-demos.html#guinterp-a-shiny-gui-to-support-spatial-interpolation"><i class="fa fa-check"></i><b>1.4.7</b> “GUInterp”: a Shiny GUI to support spatial interpolation</a></li>
<li class="chapter" data-level="1.4.8" data-path="shiny-demos.html"><a href="shiny-demos.html#a-demonstration-of-abacus-apps-based-activities-for-communicating-and-understanding-statistics"><i class="fa fa-check"></i><b>1.4.8</b> A demonstration of ABACUS: Apps Based Activities for Communicating and Understanding Statistics</a></li>
<li class="chapter" data-level="1.4.9" data-path="shiny-demos.html"><a href="shiny-demos.html#scoring-the-implicit-association-test-has-never-been-easier-dscoreapp"><i class="fa fa-check"></i><b>1.4.9</b> Scoring the Implicit Association Test has never been easier: DscoreApp</a></li>
<li class="chapter" data-level="1.4.10" data-path="shiny-demos.html"><a href="shiny-demos.html#rtrhexng-hexagon-sticker-app-for-rtrng"><i class="fa fa-check"></i><b>1.4.10</b> rTRhexNG: Hexagon sticker app for rTRNG</a></li>
</ul></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">eRum2020 Contributed Program</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="posters" class="section level2">
<h2><span class="header-section-number">1.2</span> Posters</h2>
<div id="a-flexible-dashboard-for-monitoring-platform-trials" class="section level3">
<h3><span class="header-section-number">1.2.1</span> A flexible dashboard for monitoring platform trials</h3>
<p><strong>Alessio Crippa</strong>, <em>Karolinska Institutet, postdoc</em></p>
<p>Track(s): R Applications</p>
<p><strong>Abstract:</strong></p>
<p>The Data and Safety Monitoring Board (DSMB) is an essential component for a successful clinical trial. It consists of an independent group of experts that periodically revise and evaluate the accumulating data from an ongoing trial to assess patients’ safety, study progress, and drug efficacy. Based on their evaluation, a recommendation to continue, modify or stop the trial will be delivered to the trial’s sponsor. It is essential to provide the DSMB with the best delivery visualization tools for monitoring on a regular basis the live data from the study trial.
We designed and developed an interactive dashboard using flexdashboard for R as a helping tool for assisting the DSMB in the evaluation of the results of the ProBio study, a clinical platform for improving treatment decision in patients with metastatic castrate resistant prostate cancer. We will focus on the customized structure for best displaying the most interesting variables and the adoption of interactive tools as a particularly useful aid for the assessment of the ongoing data. We will also cover the connection to the data sources, the automatic generation process, and the selected permission for the people in the DSMB to access the dashboard.</p>
<p>Coauthor(s): Andrea Discacciati, Erin Gabriel, Martin Eklund .</p>
</div>
<div id="prda-package-enhancing-statistical-inference-via-prospective-and-retrospective-design-analysis." class="section level3">
<h3><span class="header-section-number">1.2.2</span> PRDA package: Enhancing Statistical Inference via Prospective and Retrospective Design Analysis.</h3>
<p><strong>Angela Andreella</strong>, <em>University of Padua</em></p>
<p>Track(s): R Life Sciences</p>
<p><strong>Abstract:</strong></p>
<p>There is a growing recognition of the importance of power analysis and calculation of the appropriate sample size when planning a research experiment. However, power analysis is not the only relevant aspect of the design of an experiment. Other inferential risks, such as the probability of estimating the effect in the wrong direction or the average overestimation of the actual effects, are also important. The evaluation of these inferential risks as well as the statistical power, in what Gelman and Carlin (2014) defined as Design Analysis, may help researchers to make informed choices both when planning an experiment or evaluating study results.
We introduce the PRDA (Prospective and Retrospective Design Analysis) package that allows researchers to carry a Design Analysis under different experimental scenarios (Altoè et al., 2020). Considering a plausible effect size (or its prior distribution) researchers can evaluate either the inferential risks for given sample size or the required sampled size to obtain a given statistical power.
Previously, PRDA functions were limited to mean differences between groups considering Cohen’s d in the Null significance Hypothesis Testing (NHST) framework. Now, we present the newly developed features that include other effect sizes (such as Pearson’s correlation) as well as Bayes Factor hypothesis testing.</p>
<p>Coauthor(s): Vesely Anna, Zandonella Callegher Claudio, Pastore Massimiliano, Altoè Gianmarco .</p>
</div>
<div id="automate-flexdashboard-with-github" class="section level3">
<h3><span class="header-section-number">1.2.3</span> Automate flexdashboard with GitHub</h3>
<p><strong>Binod Jung Bogati</strong>, <em>Data Analyst Intern at VIN</em></p>
<p>Track(s): R Dataviz &amp; Shiny</p>
<p><strong>Abstract:</strong></p>
<p>flexdashboard is a great tool for building an interactive dashboard in R. We can host it for free on GitHub Pages, Rpubs and many other places.</p>
<p>Hosted flexdashboard is static so changes in our data we need to manually update and publish every time. If we want to auto-update we may need to integrate Shiny. However, it may not be suitable for every case.</p>
<p>To overcome this, we have a solution called GitHub Action. It’s a feature from GitHub which automates our tasks in a convenient way.</p>
<p>With the help of GitHub Actions, we can automate our flexdashboard (Rmarkdown) updates. It builds a container that runs our R scripts. We can trigger it every time we push on GitHub or schedule it every X minutes/hours/days/month.</p>
<p>If you want to learn more about the GitHub Action. And also know how to automate updates on your flexdashboard. Please do come and join me.</p>
</div>
<div id="easyreporting-a-bioconductor-package-for-reproducible-research-implementation" class="section level3">
<h3><span class="header-section-number">1.2.4</span> EasyReporting: a Bioconductor package for Reproducible Research implementation</h3>
<p><strong>Dario Righelli</strong>, <em>Department of Statistics, University of Padua, Post-Doc</em></p>
<p>Track(s): R Applications, R Life Sciences</p>
<p><strong>Abstract:</strong></p>
<p>EasyReporting is a novel R/Bioconductor package for speeding up Reproducible Research (RR) implementation when analyzing data, implementing workflows or other packages.
It is an S4 class helping developers to integrate an RR layer inside their software products, as well as data analysts speeding up their report production without learning the rmarkdown language.
Thanks to minimal additional efforts by developers, the end-user has available a rmarkdown file within all the source code generated during the analysis, divided into Code Chunks (CC) ready for the compilation. Moreover, EasyReporting gives also the possibility to add natural language comments and textual descriptions into the final report to be compiled for producing an enriched document that incorporates input data, source code and output results. Once compiled, the final document can be attached to the publication of the analysis as supplementary material, helping the interested community to entirely reproduce the computational part of work.
Despite other previously proposed solutions, that usually require a significant effort by the final user, potentially bringing him/her to renounce to include RR inside the scripts, our approach is versatile and easy to be incorporated, allowing to the final developer/analyzer to automatically create and store a rmarkdown document, and providing also methods for its compilation.</p>
<p>Coauthor(s): angelini claudia .</p>
</div>
<div id="newwave-a-scalable-r-package-for-the-dimensionality-reduction-of-single-cell-rna-seq" class="section level3">
<h3><span class="header-section-number">1.2.5</span> NewWave: a scalable R package for the dimensionality reduction of single-cell RNA-seq</h3>
<p><strong>Federico Agostinis</strong>, <em>Università degli studi di Padova, Fellowship</em></p>
<p>Track(s): R Life Sciences</p>
<p><strong>Abstract:</strong></p>
<p>The fast development of single cell sequencing technologies in the recent
years has generated a gap between the throughput of the experiments and the
capability of analizing the generated data.
One recent method for dimensionality reduction of single-cell RNA-seq data is
zinbwave, it uses zero inflated negative binomial likelihood function optimization
to find biological meaningful latent factors and remove batch effect. Zinbwave
has optimal performance but has some scalability issues due to large memory
usage. To address this, we developed an R package with new software architec-
ture extending zinbwave.
In this package, we implement mini-batch stochastic gradient descent and the
possibility of working with HDF5 files. We decide to use a negative binomial
model following the observation that droplet sequencing technologies do not
induce zero inflation in the data. Thanks to these improvements and the possi-
bility of massively parallelize the estimation process using PSOCK clusters, we
are able to speed up the computations with the same or even better results than
zinbwave. This type of parallelization can be used on multiple hardware setups,
ranging from simple laptops to dedicated server clusters. This, paired with the
ability to work with out-of-memory data, enables us to analyze datasets with
milions of cells.</p>
<p>Coauthor(s): Chiara Romualdi, Gabriele Sales, Davide Risso .</p>
</div>
<div id="orf-ordered-random-forests" class="section level3">
<h3><span class="header-section-number">1.2.6</span> orf: Ordered Random Forests</h3>
<p><strong>Gabriel Okasa</strong>, <em>Research Assistant and PhD Candidate at the Swiss Institute for Empirical Economic Research, University of St. Gallen, Switzerland</em></p>
<p>Track(s): R Machine Learning &amp; Models</p>
<p><strong>Abstract:</strong></p>
<p>The R package ‘orf’ is a software implementation of the Ordered Forest estimator as developed in Lechner and Okasa (2019). The Ordered Forest flexibly estimates the conditional class probabilities of models involving categorical outcomes with an inherent ordering structure, known as ordered choice models. Additionally to common machine learning algorithms, the Ordered Forest enables estimation of marginal effects together with statistical inference and thus provides comparable output as in standard econometric models. Accordingly, the ‘orf’ package provides generic R functions to estimate, predict, plot, print and summarize the estimation output of the Ordered Forest along with various options for specific forest-related tuning parameters. Finally, computational speed is ensured as the core forest algorithm relies on the fast C++ forest implementation from the ranger package (Wright and Ziegler 2017).</p>
<p>Coauthor(s): Michael Lechner .</p>
</div>
<div id="power-supply-health-status-monitoring-dashboard" class="section level3">
<h3><span class="header-section-number">1.2.7</span> Power Supply health status monitoring dashboard</h3>
<p><strong>Marco Calderisi</strong>, <em>Kode srl, CTO</em></p>
<p>Track(s): R Dataviz &amp; Shiny</p>
<p><strong>Abstract:</strong></p>
<p>The Primis project dashboard allows to perform an analysis on the health status of power supplies boards on two levels:
(1) analysis of a specific board, to check its status and the presence of any anomalies,
(2) analysis of multiple boards within a single Power Supply, to check if the set of boards reveals abnormal behavior and if some boards behave in a distinctly different way from the others.
The analysis algorithms and the web application were created using the programming language R, and in particular the Shiny library.
The application is therefore divided into two parts that reflect these different types of analysis, called respectively “Product View” (analysis and diagnostics of a specific board) and “Product Comparison” (comparison analysis between multiple boards of the same Power Supply). Both analyzes can be carried out on an arbitrary time interval, selectable through a special application menu.
The analysis is carried out by means of:
(1) univariate analysis, focusing on a specific parameter of one or more channels and displaying aggregate information regarding the status of the board in the entire observation period
(2) multivariate analysis, that is the application of multivariate algorithms that allows to perform an overall analysis of the board, taking into account all the variables simultaneously.</p>
<p>Coauthor(s): Jacopo Baldacci, Caterina Giacomelli, Ilaria Ceppa, Davide Massidda, Matteo Papi, Gabriele Galatolo, Francesca Giorgolo, ferdinando giordano, alessandro iovene .</p>
</div>
<div id="first-year-ict-students-dropout-predicting-with-r-models" class="section level3">
<h3><span class="header-section-number">1.2.8</span> First-year ICT students dropout predicting with R models</h3>
<p><strong>Natalja Maksimova</strong>, <em>Virumaa College of Tallinn University of Technology, lecturer</em></p>
<p>Track(s): R Machine Learning &amp; Models</p>
<p><strong>Abstract:</strong></p>
<p>The aim of this study is to find how it is possible to predict first-year ICT students dropout in one Estonian college, Virumaa College of Tallinn University of Technology (TalTech) and possibly to engage methods to decrease dropout rate. We perform three approaches of machine learning using R tools: logistic regressions, decision trees and Naive Bayes to predict. The models are computed on the basis of the TalTech study information system data. As a result, we propose a methodical approach that may be realized in practice at other institutions.
All applied methods yield high prediction with more than 85% accuracies. In the same time some influencing and non-influencing factors were found in predicting ICT students’ dropout.</p>
<p>Coauthor(s): Olga Dunajeva .</p>
</div>
<div id="benchmark-percentage-disjoint-data-splitting-in-cross-validation-for-assessing-the-skill-of-machine" class="section level3">
<h3><span class="header-section-number">1.2.9</span> Benchmark Percentage Disjoint Data Splitting in Cross Validation for Assessing the Skill of Machine</h3>
<p><strong>Olalekan Joseph Akintande</strong>, <em>University of Ibadan, Ph.D. Student</em></p>
<p>Track(s): R Machine Learning &amp; Models</p>
<p><strong>Abstract:</strong></p>
<p>The controversies surrounding dataset splitting technique and folklore of what has been or what should be, remain an open debate. Several authors (bloggers, researchers, and data scientists) in the field of machine learning and similar research areas, have proposed various arbitrary percentage disjoint dataset splitting (DDS) options for validating the skill of machine learning algorithms and by extension the appropriate percentage DDS based on cross-validation techniques. In this work, we propose benchmarks for which the percentage DDS procedure should be based. These benchmarks are founded on various training sizes (m) and serve as the basis and justification for the choice of an appropriate percentage DDS for assessing the skill of ML algorithms and related fields, on the concept of cross-validation techniques.</p>
<p>Coauthor(s): O.E. Olubusoye .</p>
</div>
<div id="integrating-professional-software-engineering-practices-in-medical-research-software" class="section level3">
<h3><span class="header-section-number">1.2.10</span> Integrating professional software engineering practices in medical research software</h3>
<p><strong>Patricia Ryser-Welch</strong>, <em>Newcastle University, Population Health Science Institute, Research Associate,</em></p>
<p>Track(s): R Applications</p>
<p><strong>Abstract:</strong></p>
<p>Health data sets are getting bigger, more complex, and are increasingly being linked up with other data sources. With this trend there is an increasing risk of patient identification and disclosure. Two different ways of mitigating this risk are to use a federated analysis approach or to use a data safe haven.</p>
<p>DataSHIELD (www.datashield.ac.uk) is an established federated data analysis tool that is used in the medical sciences. This software has a variety of methods to reduce the risk of disclosure built in. Here we describe the steps we are taking to apply modern software engineering methodologies to DataSHIELD. The upcoming Medical Devices legislation requires that software has more rigourous testing done on it. While this legislation does not directly apply to software used for research, we think it is important the ideas behind this do filter down to research software. For us these principles include testing that functions work, as well as testing that they produce the correct answers. Using a static standard data set to test against (that is publicly available) is also an important aspect. This work is being done in a continuous integraion framework using Microsoft Azure. Additionally all our software is developed as open source.</p>
<p>In addition to the protection DataSHIELD provides on its own we are also integrating it into our Trusted Research Environment as part of Connected Health Cities North East and North Cumbria. This will give an extra level of protection to data that may automatically flow from multiple data sources. Additionally, as analysis can be done in a federated way it means that that data does not need to leave its data controller’s environment. This opens up the possibility of analysis happening across trusts and regions.</p>
<p>Coauthor(s): <a href="http://www.datashield.ac.uk" class="uri">http://www.datashield.ac.uk</a> .</p>
</div>
<div id="dealing-with-changing-administrative-boundaries-the-case-of-swiss-municipalities" class="section level3">
<h3><span class="header-section-number">1.2.11</span> Dealing with changing administrative boundaries: The case of Swiss municipalities</h3>
<p><strong>Tobias Schieferdecker</strong>, <em>Daily dealings with data at cynkra</em></p>
<p>Track(s): R Applications</p>
<p><strong>Abstract:</strong></p>
<p>Switzerland’s municipalities are frequently merged or reorganized, in an attempt to reduce costs and increase efficiency.
These mergers create a substantial problem for data analysis.
Often it is desirable to study a municipality over time.
But in order to create a time series for a region of interest, its borders should stay constant.
Our goal is to provide R-functions that allow an easy and consistent handling of these mergers.
We create a mapping table for municipality ID’s for a specified period of time, that allows us to track the mergers over time.
We also provide weights, such as population as well as area of the municipalities, to facilitate the construction of weighted time series.
Various other municipality mutations are also taken into account.</p>
<p>We are creating two R packages: an infrastructure package that handles the task of keeping the data up to date; and a user package that contains the functions to deal with the mergers.</p>
<p>Coauthor(s): Thomas Knecht, Kirill Müller, Christoph Sax .</p>
</div>
<div id="baddea-an-r-package-for-measuring-firms-efficiency-adjusted-by-undesirable-outputs" class="section level3">
<h3><span class="header-section-number">1.2.12</span> badDEA: An R package for measuring firms’ efficiency adjusted by undesirable outputs</h3>
<p><strong>Yann Desjeux</strong>, <em>INRAE, France</em></p>
<p>Track(s): R Applications</p>
<p><strong>Abstract:</strong></p>
<p>Growing concerns on the detrimental effects of human production activities on the environment, e.g. air, soil and water pollution, have triggered the development of new performance indicators (including productivity and efficiency measures) accounting for such undesirable impacts. Firms can now be benchmarked not only in terms of economic performance, but also in terms of environmental performance linked to production. In the performance benchmarking literature, and more specifically the one on the non-parametric approach Data Envelopment Analysis (DEA), several methodologies have been developed to consider these impacts as undesirable (or bad) outputs. Related empirical applications in the literature, performed with various software, show that conclusions differ depending on the way these undesirable outputs are introduced. However none of these methodologies are routinely developed in R. In this context, we developed the badDEA package in order to provide a consistent and single framework where users (students, researchers, practitioners) can find the major methodologies proposed in the literature to compute efficiency measures that are adjusted by undesirable outputs. In this presentation, we will describe the aim, structure and options of the badDEA package, unfolding all the methodologies in their different variants and providing a promising tool for decision-making.</p>
<p>Coauthor(s): K Hervé DAKPO; Yann DESJEUX; Laure LATRUFFE .</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="lightning-talks.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="regular-talks.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
